dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/common/GitHubHelper.java

```diff
@@ -63,11 +63,6 @@ private static void doFetchGithubUrls(
 
         // this is a directory, so we need to query github which files are there and filter them
 
-        // URL: https://api.github.com/repos/apache/camel-k/contents/examples/kamelets/kameletbindings
-        // URL: https://api.github.com/repos/apache/camel-k/contents/examples/kamelets/kameletbindings?ref=v1.7.0
-        // https://github.com/apache/camel-k/tree/main/examples/kamelets/kameletbindings
-        // https://github.com/apache/camel-k/tree/v1.7.0/examples/kamelets/kameletbindings
-
```

Comment:
All these URLs does not exists anymore. And I have not found a specific folder of examples for Pipe/KameletBinding

Reply:
There are some examples in
https://github.com/apache/camel-kamelets-examples/
https://github.com/apache/camel-k-examples/tree/main/generic-examples/kamelets
https://github.com/apache/camel-k-examples/tree/main/kamelets

Reply:
there are examples but they are not only kameletbindings as it was in the comment

Reply:
there are no tags on camel-k-examples repository neither (and in the url provided it was using tags for some)